frontend/tests/features/wheresMyGene/wheresMyGene.test.ts

```diff
@@ -859,14 +860,18 @@ describe("Where's My Gene", () => {
   });
   describe("Cell Type Filtering", () => {
     test("Filter to multiple cell types and then clear", async ({ page }) => {
+      const CELL_TYPE_NAMES = ["B cell", "T cell", "PP cell"];
```

Comment:
keeping code DRY and use `PP cell` over `NK cell`, since `NK cell` is not available in dev/staging WMG data